site/content/en/docs/Installation/Install Agones/confirm.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Confirming Agones Installation"
```

Comment:
Since Install with Terraform is at the same level as Create Cluster and Install Agones maybe move this up a level and link it from the two terraform guides as well?

Reply:
That is an excellent suggestion. Nice! :+1: